README.md

```diff
@@ -74,6 +74,18 @@ You should expect to see a message like `Application startup complete.` in the a
 python scripts/test.py
 ```
 
+For more comprehensive testing, install pytest
+```sh
+conda activate ndif-dev
```

Comment:
Instead of assuming a package manager and instructing them how to install it, just directly link to pytest docs.